### PR TITLE
[Documentation] Improve RelayMiner staking related documentation

### PR DESCRIPTION
**Pull request by red-0ne**
## Summary

Improve supplier staking config and relay miner config documentation w.r.t upstaking, PRC types and Supplier<->RelayMiner `signing_keyname` relationship.

## Issue

The documentation was not clear enough as to
* Why upstaking is needed to update the `Supplier` provided services.
* Why it is not possible to have multiple `Supplier`s assigned to the same `RelayMiner`
* What is the purpose of having multiple `RPCType`s per `Service` provided

image

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [x] Documentation
- [ ] Other (specify)

## Testing

**Documentation changes** (only if making doc changes)
- [x] `make docusaurus_start`; only needed if you make doc changes

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [ ] I have commented my code
- [ ] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

File: docusaurus/docs/operate/configs/relayminer_config.md [hunk 1]
```diff
@@ -116,6 +116,20 @@ The name of the key that will be used to sign transactions, derive the public ke
 and the corresponding address. This key name MUST be present in the keyring that is used
 to start the `RelayMiner` instance.
 
+:::note
```
Comment (Olshansk) [inline docusaurus/docs/operate/configs/relayminer_config.md:119]: Wdyt of adding this as an example?

```mermaid
flowchart
    
    subgraph USW[US West]
        RM1["Relay Miner 1 <br> (signing_key_name=Supplier1)"]
    end

    subgraph USE[US East]
        RM2["Relay Miner 2 <br> (signing_key_name=Supplier1)"]
    end

    subgraph EUC[EU Central]
        RM3["Relay Miner 4 <br> (signing_key_name=Supplier2)"]
    end    

    subgraph CAC[CA Central]
        RM4["Relay Miner 3 <br> (signing_key_name=Supplier3)"]
    end

    subgraph PB[POKT Blockchain]
        direction TB
        S1[Supplier1]
        S2[Supplier2]
    end

    S1 -.- RM1
    S1 -.- RM2
    S1 -.- RM3
    S2 -.- RM4
```

File: docusaurus/docs/operate/configs/relayminer_config.md [hunk 2]
```diff
@@ -116,6 +116,20 @@ The name of the key that will be used to sign transactions, derive the public ke
...
+Multiple `RelayMiner`s can be configured with the same `signing_key_name` to
+sign `RelayResponse`s and submit `Claim`s and `Proof`s transactions to the Pocket
+network. (e.g. This is useful for a `Supplier` that is willing to provide redundant
+or geographically distributed services.)
+
+But it is not possible to have a single `RelayMiner` instance running with multiple
+`signing_key_name`s as this would involve more complex logic and/or configuration
+to determine which key to use, especially in the case of `Supplier`s that have
+overlapping services provided.
+
```
Comment (Olshansk) [inline docusaurus/docs/operate/configs/relayminer_config.md:130]:
```suggestion

tl;dr A 1:N Supplier:RelayMiner is okay but a N:1 Supplier:RelayMiner relationship is not.

```

File: docusaurus/docs/operate/configs/supplier_staking_config.md [hunk 3]
```diff
@@ -65,6 +65,13 @@ a new `service`, then `stake_amount: 1001upokt` should be specified in the
 configuration file. This will increase the stake by `1upokt` and deduct `1upokt`
 from the `Supplier`'s account balance.
 
+The upstaking requirement is to ensure that a `Supplier` incurs a cost for
+changing the services they provide too frequently, which could lead to a poor user
+experience for `Gateways` and `Applications`.
+
+The upstake amount may be adjusted in the future based on network conditions
+and feedback from the community.
+
```
Comment (Olshansk) [inline docusaurus/docs/operate/configs/supplier_staking_config.md:74]:
```suggestion

```

File: docusaurus/docs/operate/configs/supplier_staking_config.md [hunk 4]
```diff
@@ -65,6 +65,13 @@ a new `service`, then `stake_amount: 1001upokt` should be specified in the
 configuration file. This will increase the stake by `1upokt` and deduct `1upokt`
 from the `Supplier`'s account balance.
 
+The upstaking requirement is to ensure that a `Supplier` incurs a cost for
+changing the services they provide too frequently, which could lead to a poor user
+experience for `Gateways` and `Applications`.
```
Comment (Olshansk) [inline docusaurus/docs/operate/configs/supplier_staking_config.md:70]:
```suggestion
experience for `Gateways` and `Applications`. It is also necessary to dissuade sybil
or flood attacks on the network.
```

**Review by Olshansk — CHANGES_REQUESTED** [review]
A few comments / suggestions but looks great otherwise

File: docusaurus/docs/operate/configs/supplier_staking_config.md (hunk 3 above)
Comment (red-0ne) [reply docusaurus/docs/operate/configs/supplier_staking_config.md:74]: Would love to know why exactly 1upokt then.

**Review by Olshansk — CHANGES_REQUESTED** [review]

File: docusaurus/docs/operate/configs/relayminer_config.md [hunk 5]
```diff
@@ -116,6 +116,54 @@ The name of the key that will be used to sign transactions, derive the public ke
...
 to start the `RelayMiner` instance.
 
+:::note
+
+Multiple `RelayMiner`s can be configured with the same `signing_key_name` to
+sign `RelayResponse`s and submit `Claim`s and `Proof`s transactions to the Pocket
+network. (e.g. This is useful for a `Supplier` that is willing to provide redundant
+or geographically distributed services.)
+
+```mermaid
```
[the remark this answers is not in the document]
Comment (red-0ne) [reply docusaurus/docs/operate/configs/relayminer_config.md:126]: Already did. Did it render wrong for you?

[the remark this answers is not in the document]
Comment (red-0ne) [reply docusaurus/docs/operate/configs/relayminer_config.md:126]: image

[the remark this answers is not in the document]
Comment (Olshansk) [reply docusaurus/docs/operate/configs/relayminer_config.md:126]: Nice!

**Review by Olshansk — APPROVED** [review]
